client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -180,16 +206,26 @@ class CancelPurchase extends React.Component {
 					<div className="cancel-purchase__refund-amount">
 						{ this.renderFooterText( this.props ) }
 					</div>
-					<CancelPurchaseButton purchase={ purchase } selectedSite={ this.props.selectedSite } />
+					<CancelPurchaseButton
+						purchase={ purchase }
+						includedDomainPurchase={ this.props.incudedDomainPurchase }
+						disabled={ this.state.cancelBundledDomain && ! this.state.confirmCancelBundledDomain }
+						selectedSite={ this.props.selectedSite }
+						cancelBundledDomain={ this.state.cancelBundledDomain }
+					/>
 				</CompactCard>
 			</Main>
 		);
 	}
 }
 
-export default connect( ( state, props ) => ( {
-	hasLoadedSites: ! isRequestingSites( state ),
-	hasLoadedUserPurchasesFromServer: hasLoadedUserPurchasesFromServer( state ),
-	selectedPurchase: getByPurchaseId( state, props.purchaseId ),
-	selectedSite: getSelectedSiteSelector( state ),
-} ) )( localize( CancelPurchase ) );
+export default connect( ( state, props ) => {
+	const purchase = getByPurchaseId( state, props.purchaseId );
+	return {
+		hasLoadedSites: ! isRequestingSites( state ),
+		hasLoadedUserPurchasesFromServer: hasLoadedUserPurchasesFromServer( state ),
+		selectedPurchase: getByPurchaseId( state, props.purchaseId ),
```

Comment:
I think this is the same as the above call and could just be replaced by `purchase` here.